plugins/woocommerce/tests/performance/bin/init-sample-products.sh

```diff
@@ -32,9 +32,19 @@ wp-env run tests-cli wp import wp-content/plugins/woocommerce/sample-data/sample
 # install Storefront
 wp-env run tests-cli wp theme install storefront --activate
 
-if [ $ENABLE_HPOS == 1 ]; then
-	echo -e 'Enable High-Performance Order Tables\n'
-	wp-env run tests-cli wp plugin install https://gist.github.com/vedanshujain/564afec8f5e9235a1257994ed39b1449/archive/b031465052fc3e04b17624acbeeb2569ef4d5301.zip --activate
+# Enable HPOS
```

Comment:
Good point! Thanks. I removed the redundant comments in this commit - e5a9b21a61f13ddffdfeeac8acfd3a83c93d2359

As for the reason why I placed those comments, I was only following the pattern from the other sections, like the ones shown here:
<img width="710" alt="NqKrdaSxVk" src="https://github.com/woocommerce/woocommerce/assets/4509348/43dea0bf-bf84-4afb-b740-fc6d35047311">

However, since the HPOS lines have their corresponding `echo` printouts anyway, then I agree with you that it's ok to remove the comments as they're already being redundant.